software/evershop.yml

```
@@ -0,0 +1,12 @@
+name: EverShop
+website_url: https://evershop.io/
+description: EverShop is a NodeJS ecommerce platform with essential commerce features. Built with NodeJS and React. Modular architecture and fully customizable.
+licenses:
+  - GPL-3.0
+platforms:
+  - Nodejs
```

```suggestion
platforms:
  - Docker
  - Nodejs
```

Also [supports Docker](https://evershop.io/docs/development/getting-started/installation-guide#install-evershop-using-docker) as an installation method

---

Thanks. Updated